utils/kind-with-registry.sh

```diff
@@ -54,7 +54,7 @@ fi
 echo "Registry Host: ${reg_host}"
 
 # create a cluster with the local registry enabled in containerd
-cat <<EOF | kind create cluster --name "${KIND_CLUSTER_NAME}" --config=-
+cat <<EOF | kind create cluster --image kindest/node:v1.21.2 --name "${KIND_CLUSTER_NAME}" --config=-
```

Comment:
Version should be set as configurable.

Reply:
I added it in my env for installation. This is not the feature of this PR. 
Will add another PR for this.